recipe-page/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Practice
+.my-secret-keys
```

Comment:
Again with the secrets. Always so secret. Secret this, secret that, you'd swear we were in grade school with all the secrets!